ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -0,0 +1,624 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: IMAGE_INSTALLER
+  required: true
+- name: IMAGE_TESTS
+  required: true
+- name: LOCAL_IMAGE_UPI_INSTALLER
+  required: true
+- name: CLUSTER_TYPE
+  required: true
+- name: TEST_COMMAND
+  required: true
+- name: RELEASE_IMAGE_LATEST
+  required: true
+- name: BASE_DOMAIN
+  value: origin-ci-int-aws.dev.rhcloud.com
+  required: true
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+  - kind: SystemGroup
+    name: system:authenticated
+
+# Give edit access to a known bot
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-namespace-editors
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: edit
+  subjects:
+  - kind: ServiceAccount
+    namespace: ci
+    name: ci-chat-bot
+
+# Route for boostrap ignition file
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: ${JOB_NAME_SAFE}-bootstrap-exporter
+    namespace: ${NAMESPACE}
+  spec:
+    host: ${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.svc.ci.openshift.org
+    to:
+      name: ${JOB_NAME_SAFE}-bootstrap-exporter
+    tls:
+      termination: Edge
+      insecureEdgeTerminationPolicy: Redirect
+
+# Service for ignition file
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${JOB_NAME_SAFE}-bootstrap-exporter
+    namespace: ${NAMESPACE}
+  spec:
+    selector:
+      app: testpod
+    ports:
+    - port: 80
+      targetPort: 8080
+
+# The e2e pod spins up a cluster, runs e2e tests, and then cleans up the cluster.
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown
+      ci-operator.openshift.io/save-container-logs: "true"
+      ci-operator.openshift.io/container-sub-tests: "setup,test,artifacts,teardown"
```

Comment:
Can we drop `artifacts` here to match the rest of this directory:

```console
$ git grep 'sub-tests.*artifacts' origin/pr/3305 -- ci-operator/templates/openshift/installer
origin/pr/3305:ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml:      ci-operator.openshift.io/container-sub-tests: "setup,test,artifacts,teardown"
```

(although personally I'm not clear on how artifacts get uploaded after being collected from the pod ;)

Reply:
ci-operator would store things from `artifacts` volume